### PR TITLE
Admin repo search and admin view lex schemas

### PR DESCRIPTION
**Pull request by devinivy**
This work adds an admin-only method `com.atproto.admin.searchRepos` for searching through all repos and understanding their moderation status.  This also contains proposed lex schemas for a variety of upcoming admin views of moderation reports, actions, repos, and records.

Along the way I moved some things around, notably breaking `atproto/admin.ts` out into a directory filled with separate files per xrpc method.

### CODE REVIEW

**Comment by pfrazee** [comment]
Sweet, LGTM

File: packages/pds/src/api/com/atproto/admin/resolveModerationReports.ts [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   TAKEDOWN,
-} from '../../../lexicon/types/com/atproto/admin/moderationAction'
-import { ModerationAction } from '../../../db/tables/moderation'
-import { InputSchema as ActionInput } from '../../../lexicon/types/com/atproto/admin/takeModerationAction'
+} from '../../../../lexicon/types/com/atproto/admin/moderationAction'
+import { ModerationAction } from '../../../../db/tables/moderation'
+import { InputSchema as ActionInput } from '../../../../lexicon/types/com/atproto/admin/takeModerationAction'
 
 export default function (server: Server, ctx: AppContext) {
   server.com.atproto.admin.takeModerationAction({
```
Comment (dholms) [inline packages/pds/src/api/com/atproto/admin/resolveModerationReports.ts:15]: think this one & resolveModerationReport got mixed up files

File: packages/pds/src/services/moderation/index.ts [hunk 2]
```diff
@@ -22,6 +23,8 @@ export class ModerationService {
     return (db: Database) => new ModerationService(db, messageQueue, blobstore)
   }
 
+  views = new ModerationViews(this.db)
```
Comment (dholms) [inline packages/pds/src/services/moderation/index.ts:26]: nice like this pattern

**Review by dholms — APPROVED** [review]
looks good! mismatched xrpc methods were the only things i saw

File: packages/pds/src/api/com/atproto/admin/resolveModerationReports.ts (hunk 1 above)
Comment (devinivy) [reply packages/pds/src/api/com/atproto/admin/resolveModerationReports.ts:15]: Oops, thanks for catching that!

**Pull request merged**